notebooks/galaxy_redshift/redshift_fitting.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>Can you please include the following details to make sure that the notebook is consistent with others: the use case, the type of data, the tools used, and any cross-instrument usage, if applicable?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>